static/app/views/dashboardsV2/widgetCard/widgetQueries.tsx

```diff
@@ -142,7 +149,8 @@ class WidgetQueries extends React.Component<Props, State> {
       !isEqual(widget.displayType, prevProps.widget.displayType) ||
       !isEqual(widget.interval, prevProps.widget.interval) ||
       !isEqual(widgetQueries, prevWidgetQueries) ||
-      !isSelectionEqual(selection, prevProps.selection)
+      !isSelectionEqual(selection, prevProps.selection) ||
+      !isEqual(cursor, prevProps.cursor)
```

Comment:
do we need a deep comparison here?

Reply:
probably not necessary, I've updated to just `!==`!